README.md

```diff
@@ -89,31 +89,32 @@ USAGE: inspec_tools generate_inspec_metadata
 
 ## Docker Usage
 
-Building the Docker container and running it with `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name>` will print a listing of all the command with a short description.
-For detailed help on any command, run `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name> help [COMMAND]`. Help can also be called with the `-h, --help` flags after any command, like `docker run -v`pwd`:/share -e CHEF_LICENSE=accept <image name> xccdf2inspec -h`.
+Building the Docker container and running it with `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name>` will print a listing of all the command with a short description.
+For detailed help on any command, run `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name> help [COMMAND]`. Help can also be called with the `-h, --help` flags after any command, like `docker run -v` `` ` ``pwd`` ` `` `:/share -e CHEF_LICENSE=accept <image name> xccdf2inspec -h`.
+`` test` ``
 
 ### generate_map
```

Comment:
I'm not sure we need a section for every single command duplicated twice in the README. The README should just effectively say "When using the Docker container, run ```docker run -itv`pwd`:/share -e CHEF_LICENSE=accept mitre/inspec_tools [COMMAND]``` where `[COMMAND]` is any of the Command line Usage options listed above."

Reply:
Also, the container name can be replaced with `mitre/inspec_tools` since that will be the container name.